csrc/attention/dtype_float32.cuh

```diff
@@ -86,6 +86,14 @@ inline __device__ float4 add(float4 a, float4 b) {
   return c;
 }
 
+// for compiling, the above function seems to be useless
```

Comment:
I am not sure the comment is right. Delete it?

Reply:
ok.